translations/messages+intl-icu.bg.xlf

```diff
@@ -20,15 +20,15 @@
             </trans-unit>
             <trans-unit id="http_error.name">
                 <source>http_error.name</source>
-                <target>Грешка %status_code%</target>
+                <target>Грешка {status_code, number}</target>
```

Comment:
I'm not sure if the type is needed here! I added it by principe but feedback will be appreciated